engine/src/main/java/org/terasology/engine/network/PingStockComponent.java

```diff
@@ -15,7 +15,7 @@
  * <p>
  * Might be used to stock ping information and display it in future.
  */
-public final class PingStockComponent implements Component {
+public final class PingStockComponent implements Component<PingStockComponent> {
 
     // TODO Map<EntityRef,Long> is not supported for replication (no type handler),
```

Comment:
We do have https://github.com/MovingBlocks/Terasology/blob/develop/subsystems/TypeHandlerLibrary/src/main/java/org/terasology/persistence/typeHandling/coreTypes/GenericMapTypeHandler.java for arbitrary maps by now - can this component be simplified now?